build/mitre_attack_checker/falco_mitre_attack_checker/tests/resources/falco_rules_test.yaml

```diff
@@ -0,0 +1,88 @@
+- required_engine_version: 17
```

Comment:
Could you rebase this PR and change this to the new semver versioning?

Reply:
Indeed ! I am fixing it.

Reply:
done !